src/App.tsx

```diff
@@ -1,24 +1,13 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import Counter from "./components/Counter";
+import Hello from "./components/Hello";
+import SearchForm from "./components/Search/SearchForm";
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.tsx</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div>
+      <Hello></Hello>
+      <Counter></Counter>
+      <SearchForm></SearchForm>
```

Comment:
[Q] I see all these 3 components do not have a [child prop](https://reactjs.org/docs/composition-vs-inheritance.html#containment) specified. Can we simply use [self-closing tags](https://reactjs.org/docs/introducing-jsx.html#specifying-children-with-jsx)?

Reply:
Updated as self-closing.